api-server/web-server/src/api/v1.rs

```diff
@@ -287,10 +309,35 @@ async fn get_transaction(
         ))
 }
 
-#[allow(clippy::unused_async)]
+pub async fn submit_transaction<T: ApiServerStorage>(
+    State(state): State<ApiServerWebServerState<Arc<T>, Option<Arc<impl TxSubmitClient>>>>,
+    body: String,
+) -> Result<impl IntoResponse, ApiServerWebServerError> {
+    let tx = HexEncoded::<SignedTransaction>::from_str(&body)
+        .map_err(|_| {
+            ApiServerWebServerError::ClientError(
+                ApiServerWebServerClientError::InvalidSignedTransaction,
+            )
+        })?
+        .take();
+
+    state
+        .rpc
+        .expect("must be present if route is enabled")
+        .submit_tx(tx)
+        .await
+        .map_err(|e| {
+            ApiServerWebServerError::ServerError(ApiServerWebServerServerError::RpcError(
+                e.to_string(),
+            ))
```

Comment:
Please use http code 401 or 403 for this.

Reply:
I think I confused RPC error with whatever exists due to the router not working. I'll resolve this and ask elsewhere.